### PR TITLE
Add end-to-end to be run against Alexander testnet

### PR DESCRIPTION
**Pull request by philipstanislaus**

### CODE REVIEW

**Comment by philipstanislaus** [comment]
@vimukthi-git can we merge this one? 

Thanks!

File: teste2e/main_test.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import (
+	"fmt"
+	"testing"
+
+	gsrpc "github.com/centrifuge/go-substrate-rpc-client"
+	"github.com/centrifuge/go-substrate-rpc-client/config"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetBlockHashAndVersion(t *testing.T) {
```
Comment (vimukthi-git) [inline teste2e/main_test.go]: are we not testing the metadata call?

File: client/client.go [hunk 2]
```diff
@@ -45,6 +46,11 @@ type client struct {
 	//metadataLock sync.RWMutex
 }
 
+// Returns the URL the client connects to
+func (c client) GetURL() string {
```
Comment (vimukthi-git) [inline client/client.go]: in go getters shall not contain `Get` by convention (For RPCs its ok as they are not getters, but network calls mapping to rpc calls).
```suggestion
func (c client) URL() string {
```

File: teste2e/main_test.go (hunk 1 above)
Comment (pstehlik) [reply teste2e/main_test.go]: does it matter for this specific one?

Comment (vimukthi-git) [reply teste2e/main_test.go]: I thought its part of the deliverable?

Comment (vimukthi-git) [reply teste2e/main_test.go]: Ok ignore this, I see it's calling `state.getVersion` which tests metadata :) 
sorry for the confusion.

**Pull request merged**